department-of-veterans-affairs/abd-vro
# MCP 2445: Python assessment code cleanup

## What was the problem?

Some things were not named clearly. Main function needed to be cleaned up. 

Associated tickets or Slack threads:

- [MCP-2445](https://amida.atlassian.net/browse/MCP-2445)

## How does this fix it?[^1]

Renaming. Allow conditions without a code and just set a default to empty string. 

Not included in this PR:
`log instances of data that we strip out for forensics purposes`

## How to test this PR
- Step 1 Run unit tests
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
